pkg/cmd/server/origin/master.go

```diff
@@ -1063,6 +1067,23 @@ func (c *MasterConfig) RunSecurityAllocationController() {
 	controller.Run()
 }
 
+func (c *MasterConfig) ensureDefaultSecurityContextConstraints() {
```

Comment:
just realized where this function was... move it up with the other ensure* functions